### ISSUE TITLE
Keep popup open after pasting

### ISSUE THREAD

**Issue by rcw5** [p0deje/Maccy#1001]
### Before Submitting Your Feature Request

- [X] Check that there isn't already a similar feature request to avoid creating a duplicate.
- [ ] I have seen the [FAQ](https://github.com/p0deje/Maccy?tab=readme-ov-file#faq).

### Problem

I want to copy multiple text snippets from Maccy into a document. The Maccy popup closes automatically after every paste, so I need to reopen the Maccy UI for each piece of text. Having to reopen Maccy each time slows down my workflow.

### Solution

It would be great if there was a way to keep the Maccy popup open after pasting. This would make my workflow much simpler as there would be no need to reopen Maccy before pasting each piece of text.

This could be exposed via a pin on the UI - if enabled, Maccy will stay open after pasting, if disabled it would close automatically

Thanks for the great app!

**Comment by louisgavalda** [comment]
Yes, I am also looking for a way to paste several items saved by Maccy in the fastest way. Any other option?

**Comment by p0deje** [comment]
Please see #239. We should have support for multiple pastes relatively soon (#1206).
As for keeping the pop-up open, this is possible to implement, but I wonder if it's going to be needed after the stack is done.
